root/components/common-macros.tt

```diff
@@ -834,9 +834,9 @@ END -%]
     };
     IF medium.name != '';
         args.medium_name = isolate_text(medium.name);
-        l('{medium_format} {position}: {medium_name}', args);
+        l('<span>{medium_format} {position}:</span> {medium_name}', args);
```

Comment:
I cleaned CSS rules and reduced changes made to localized messages in updated  https://github.com/metabrainz/musicbrainz-server/pull/481/commits/83e0fc444446f75bd816d9d6a2d251c3663cc5aa
I added `medium-name` class only which is useful to apply _normal_ _white-space_ wrapping style .
Other classes can be added later on if necessary without changing localized messages.